tasks/condition/condition.js

```diff
@@ -23,6 +20,15 @@
 {
     const x = 3;
     const y = 7;
+// Task 1
+if (x && y < 10) {
+  console.log(x + y);
+} else {
+  console.log(x * y);
+}
+// Task 2
+console.log(x && y < 10 ? x + y : x * y);
```

Comment:
it's okay, but you'd better use this construction for consistency purposes:
x && y < 10 ? console.log(x+y) : console.log(x * y);